config/create-common.toml

```diff
@@ -3,164 +3,137 @@
 
 	#
 	#Modify Create's impact on your terrain
-	[worldgen.v1]
+	[worldgen.v2]

```

Comment:
This configs needs to be adjusted
to match the old settings,
using the new format!